tools/build-neatvnc.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+# Update and install dependencies
+apt update
+apt install -y meson ninja-build libpixman-1-dev zlib1g-dev libdrm-dev pkgconf
+
+CURRENT_DIR=$(pwd)
+# Clone and build aml
+git clone https://github.com/any1/aml
+cd aml
+meson -Dprefix=$CURRENT_DIR/_static build
+meson configure build --default-library=static
+ninja -C build
+ninja -C build install
+
+export PKG_CONFIG_PATH=$CURRENT_DIR/_static/lib/x86_64-linux-gnu/pkgconfig
+
+# Clone and build NeatVNC
+cd ..
+git clone https://github.com/any1/neatvnc
+cd neatvnc
+git checkout v0.8
```

Comment:
Replace the above command with `git clone https://github.com/any1/neatvnc --depth=1 -b v0.8.1 && cd neatvnc`